packages/runtime-common/indexer.ts

```diff
@@ -367,8 +367,6 @@ export class Indexer {
       ['i.realm_url = ', param(realmURL.href)],
       ['i.type =', param('instance')],
       ['is_deleted = FALSE OR is_deleted IS NULL'],
-      // our tests assert that the index card should not come back in the search results, so:
-      ['url !=', param(new RealmPaths(realmURL).fileURL('index.json').href)],
```

Comment:
I didn't find an assertion that actually demands this. The tests continue to pass without it. Does anyone know of a case where we wouldn't want the index card instance in the search results?

Reply:
i think this is super old. lets get rid of it

Reply:
but let's make sure to check that the index card does not include itself in the grid.